### PR TITLE
[utils, xlucene-parser] Update @turf

### PR DESCRIPTION
**Pull request by busma13**
This PR makes the following changes:
- utils package updates:
  - add devDependency:
        @types/geojson version ^7946.0.14 
  - update dependencies:
        @turf/bbox from 6.4.0 to 7.1.0
        @turf/bbox-polygon from 6.4.0 to 7.1.0
        @turf/boolean-contains from 6.4.0 to 7.1.0
        @turf/boolean-disjoint from 6.4.0 to 7.1.0
        @turf/boolean-equal from 6.4.0 to 7.1.0
        @turf/boolean-intersect from ^6.4.0 to 7.1.0
        @turf/boolean-point-in-polygon from 6.4.0 7.1.0
        @turf/boolean-within from 6.4.0 to 7.1.0
        @turf/circle from 6.4.0 to 7.1.0
        @turf/helpers from 6.2.0 to 7.1.0
        @turf/invariant from 6.2.0 to 7.1.0
        @turf/line-to-polygon from 6.4.0 to 7.1.0

- xlucene-parser package updates:
        @turf/invariant 6.2.0 from to 7.1.0
        @turf/random from 6.4.0 to 7.1.0

### CODE REVIEW

File: packages/utils/src/geo.ts [hunk 1]
```diff
@@ -29,12 +29,15 @@ import {
     multiPolygon,
     polygon as tPolygon,
     point as tPoint,
+
```
Comment (godber) [inline packages/utils/src/geo.ts:32]:
```suggestion
```

Comment (godber) [reply packages/utils/src/geo.ts:32]: weird empty line

**Review by godber — CHANGES_REQUESTED** [review]
get rid of empty line

**Review by jsnoble — APPROVED** [review]
LGTM

**Review by godber — APPROVED** [review]

**Pull request merged**